WordPress/openverse
# Fix Tailwind line heights

## Fixes




Fixes #947 by @obulat 
Fixes #477 by @dhruvkb 

## Description



This PR moves the line height settings to `extend` and adjust them so that only the set values are used, and not the Tailwind default fallback. This means that a lot of snapshots need to be updated because in many places, the line height was set to 1.375 or 1.25 instead of 1.3 instead of 1.3 as in Openverse mockups.
You can see the current values for named line heights in the Tailwind viewer: https://wordpress.github.io/openverse/tailwind/#Line%20Height (Compare to this PR: https://wordpress.github.io/openverse/_preview/946/tailwind/#Line%20Height)

<img width="417" alt="Screenshot 2023-03-19 at 7 30 33 AM" src="https://user-images.githubusercontent.com/15233243/226153753-3527441b-6152-4409-83b5-7090bc37f7a5.png">

This PR is important before the work on Core UI improvements because the line height of the buttons is also affected. We could also add this to the scope of Core UI improvements project.

## Testing Instructions


Check the snapshots - they should look fairly similar, but the heights should change slightly. Everything should build fine (no errors of `class` not found for tailwind build step)

## Checklist



      `Update index.md`).
      a parent feature branch.
      errors.

[best_practices]:
  https://git-scm.com/book/en/v2/Distributed-Git-Contributing-to-a-Project#_commit_guidelines

## Developer Certificate of Origin



<details>
<summary>Developer Certificate of Origin</summary>

```
Developer Certificate of Origin
Version 1.1

Copyright (C) 2004, 2006 The Linux Foundation and its contributors.
1 Letterman Drive
Suite D4700
San Francisco, CA, 94129

Everyone is permitted to copy and distribute verbatim copies of this
license document, but changing it is not allowed.


Developer's Certificate of Origin 1.1

By making a contribution to this project, I certify that:

(a) The contribution was created in whole or in part by me and I
    have the right to submit it under the open source license
    indicated in the file; or

(b) The contribution is based upon previous work that, to the best
    of my knowledge, is covered under an appropriate open source
    license and I have the right under that license to submit that
    work with modifications, whether created in whole or in part
    by me, under the same open source license (unless I am
    permitted to submit under a different license), as indicated
    in the file; or

(c) The contribution was provided directly to me by some other
    person who certified (a), (b) or (c) and I have not modified
    it.

(d) I understand and agree that this project and the contribution
    are public and that a record of the contribution (including all
    personal information I submit with it, including my sign-off) is
    maintained indefinitely and may be redistributed consistent with
    this project or the open source license(s) involved.
```

</details>
